all.json

```diff
@@ -2227,6 +2228,7 @@
     "sienna.medium.com.ru",
     "sienna.polkastarter.com.es",
     "silvertechsolutions.net",
+    "sites.google.com",
```

Comment:
Should not have merged this - it blocks all there.

Reply:
I'll create a PR to remove this one then